packages/xo-server-sdn-controller/src/index.js

```diff
@@ -49,6 +50,10 @@ export const configurationSchema = {
 
 // =============================================================================
 
+const noop = () => {}
```

Comment:
No need to create a new function :slightly_smiling_face: 

```suggestion
const noop = Function.prototype
```